packages/react/package.json

```diff
@@ -89,6 +89,7 @@
     "eslint-plugin-prettier": "^5.1.3",
     "eslint-plugin-react": "^7.33.2",
     "eslint-plugin-react-hooks": "^4.6.0",
+    "fake-indexeddb": "^6.0.0",
```

Comment:
Shouldn't this be in the `devDependencies` instead?

Reply:
To me it looks like it already is. 
![image](https://github.com/user-attachments/assets/0e62fd65-1ab6-4fcf-8d43-f9f054e516e2)

Reply:
ping @mrTuomoK

Reply:
True, somehow I goi it wrong 👍